internal/cloudsql/instance.go

```diff
@@ -180,18 +183,24 @@ type ConnectionInfo struct {
 // NewConnectionInfo initializes a ConnectionInfo struct.
 func NewConnectionInfo(
 	cn instance.ConnName,
+	dnsName string,
```

Comment:
Incorporate names and structure comments from above.

Reply:
Thanks. Done, except the `dnsName` which I am not sure.